### ISSUE TITLE
Please extend documentation with relevant info

### ISSUE THREAD

**Issue by bmuel** [MicrosoftDocs/power-automate-docs#1178]
This is a very interesting and important feature.
But the documentation is too basic as it is now.

In my opinion, following information is missing:

- In case of an SPN user (is this the same as app registration?): What property has to be entered into the HTTP trigger?
- What is required to call the HTTP trigger? The article states that tenant id, object id or user id claims have to be present. In my example, this is all the case, but I still get "One or more claims either missing or does not match with the open authentication access control policy." It would be nice to have one or multiple examples how clients can call on the power automate flow. Also in different scenarios for real users and for app registration spn users.

Many thanks

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: c90fd90c-51fb-27b8-9e4c-429b2ca0075c
* Version Independent ID: c90fd90c-51fb-27b8-9e4c-429b2ca0075c
* Content: [Add OAuth authentication for HTTP request triggers - Power Automate](https://learn.microsoft.com/en-us/power-automate/oauth-authentication?tabs=classic-designer)
* Content Source: [articles/oauth-authentication.md](https://github.com/MicrosoftDocs/power-automate-docs/blob/main/articles/oauth-authentication.md)
* Service: **power-automate**
* Sub-service: **cloud-flow**
* GitHub Login: @rakrish84
* Microsoft Alias: **rakrish**

**Comment by AlexFilipovici** [comment]
@bmuel did you manage to figure out a way to pass the correct token?

**Comment by bmuel** [comment]
@AlexFilipovici no, I think it doesn't really work as proposed. 
I opened an MS support case in the meantime, but it's still ongoing.

In the meantime they extended the documentation and mentioned the required claims. But even with those and the correct audience it doesn't work for me.

**Comment by AlexFilipovici** [comment]
@bmuel the tricky part was to use the double slash in //.default. i.e. [service.flow.microsoft.com//.default](https://service.flow.microsoft.com//.default)

But anyone has a good explanation on why? I've didn't see it mentioned anywhere.

**Comment by bmuel** [comment]
@AlexFilipovici you're absolutely right. I can confirm that it works the way you described. How on earth did you uncover this?
I think there is no good explanation as to why. For other services such das Dataverse it works with /.default.

Many thanks for finding a solution to an unnecessary problem :-).
